runtime-decompiler/pom.xml

```diff
@@ -70,6 +70,20 @@
             </plugin>
         </plugins>
             <resources>
+                <resource>
```

Comment:
The commit message si wrong - the plugin wrappers and jsons are now packed in resulting jar